src/app/components/PortraitVideoCarousel/PortraitVideoPromo/index.test.tsx

```diff
@@ -3,17 +3,55 @@ import { render } from '#app/components/react-testing-library-with-providers';
 import PortraitVideoPromo from '.';
 
 describe('PortraitVideoPromo', () => {
-  it('Should contain a h3 level title', () => {
+  it('Should contain a title', () => {
```

Comment:
Do we need a test like this? I don't see any branching logic for when a title should show, so not sure why we need a unit test for it?